app/controllers/posts_controller.rb

```diff
@@ -1,69 +1,77 @@
 class PostsController < ApplicationController
-  before_action :set_post, only: %i[show edit update destroy]
+  before_action :set_user, only: %i[show]
 
-  # GET /posts or /posts.json
+  # # GET /posts or /posts.json
   def index
-    @posts = Post.all
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
   end
 
-  # GET /posts/1 or /posts/1.json
-  def show; end
-
-  # GET /posts/new
-  def new
-    @post = Post.new
+  # # GET /posts/1 or /posts/1.json
+  def show
+    @post = Post.find(params[:id])
   end
 
-  # GET /posts/1/edit
-  def edit; end
+  # # GET /posts/new
+  # def new
+  #   @post = Post.new
+  # end
 
-  # POST /posts or /posts.json
-  def create
-    @post = Post.new(post_params)
+  # # GET /posts/1/edit
+  # def edit; end
 
-    respond_to do |format|
-      if @post.save
-        format.html { redirect_to post_url(@post), notice: 'Post was successfully created.' }
-        format.json { render :show, status: :created, location: @post }
-      else
-        format.html { render :new, status: :unprocessable_entity }
-        format.json { render json: @post.errors, status: :unprocessable_entity }
-      end
-    end
-  end
+  # # POST /posts or /posts.json
+  # def create
+  #   @post = Post.new(post_params)
 
-  # PATCH/PUT /posts/1 or /posts/1.json
-  def update
-    respond_to do |format|
-      if @post.update(post_params)
-        format.html { redirect_to post_url(@post), notice: 'Post was successfully updated.' }
-        format.json { render :show, status: :ok, location: @post }
-      else
-        format.html { render :edit, status: :unprocessable_entity }
-        format.json { render json: @post.errors, status: :unprocessable_entity }
-      end
-    end
-  end
+  #   respond_to do |format|
+  #     if @post.save
+  #       format.html { redirect_to post_url(@post), notice: 'Post was successfully created.' }
+  #       format.json { render :show, status: :created, location: @post }
+  #     else
+  #       format.html { render :new, status: :unprocessable_entity }
+  #       format.json { render json: @post.errors, status: :unprocessable_entity }
```

Comment:
- You have done great so far :+1: :+1: Your app meets all the requirements and works as expected. but there are a few things you need to correct before moving to the next milestone. :smile: :wave: 

- [ ] Following the best practices, you are not meant to leave unused code as a comment on your code base. it makes your codebase look unprofessional. I'll suggest you remove all these commented-out codes. :smile: :wave:

Reply:
Implemented